charts/humio-core/templates/statefulset.yaml

```diff
@@ -165,13 +165,15 @@ spec:
             - name: AUTO_CREATE_USER_ON_SUCCESSFUL_LOGIN
               value: {{ .Values.samlConfig.autoCreateUserOnSuccessfulLogin | quote }}
             {{- end }}
-            {{- if .Values.github.enabled }}
```

Comment:
We want to keep this. Perhaps this got removed inadvertently and requires a rebase from master?